samples/auth/src/main/res/values/strings.xml

```diff
@@ -48,8 +48,8 @@
 
     <!-- Facebook -->
     <string name="facebook_app_id">your facebook app id</string>
-    <string name="fb_login_protocol_scheme">your facebook scheme</string>
-
+    <string name="fb_login_protocol_scheme">your facebook app scheme</string>
+    <string name="facebook_client_token">your facebook app token</string>
```

Comment:
Make sure we update the documentation for this update: https://backstage.forgerock.com/docs/sdks/latest/how-to/social-login/setup-android.html